R/rwebppl.R

```diff
@@ -343,21 +344,20 @@ run_webppl <- function(program_code = NULL, program_file = NULL, data = NULL,
   if (file.exists(error_file)) {
     file.remove(error_file)
   }
+  finish_file <- sprintf("/tmp/webppl_finished_%s", uid)
+  finish_arg <- finish_file
+  if (file.exists(finish_file)) {
+    file.remove(finish_file)
```

Comment:
since finish_file is randomly generated each time using a uid, will it ever already exist? Same goes for output_file. I wasn't sure what this cleaning was for (if we decide we still need it, it'd be nice to move these three things into a helper function called `clean_tmp` or something)